cmd/validate/image_test.go

```diff
@@ -1194,6 +1194,7 @@ func TestEvaluatorLifecycle(t *testing.T) {
 		fmt.Sprintf(`{"publicKey": %s, "sources": [%s]}`, utils.TestPublicKeyJSON, strings.Join(sources, ", ")),
 		"--effective-time",
 		effectiveTimeTest,
+		"--ignore-rekor",
```

Comment:
FYI: This is fine, of course. An alternative is to call `utils.SetTestRekorPublicKey(t)`.